DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml

```diff
@@ -51,8 +53,7 @@
                      month-id="Month"
                      year-id="Year"
                      css-class="nhsuk-u-margin-bottom-4"
-                     key="1"

```

Comment:
Do you know what this key was for?

Reply:
Not sure, but the date input component doesn't have a key attribute so it was preventing the input from displaying at all on this page. I've tested it and it works properly without it :)